runtime/rastrace/method_trace.c

```diff
@@ -476,14 +476,13 @@ traceMethodArgObject(J9VMThread *thr, UDATA* arg0EA, char* cursor, UDATA length)
 	} else {
 		J9Class *clazz = J9OBJECT_CLAZZ(thr, object);
 		J9JavaVM *vm = thr->javaVM;
+		const unsigned int methodStrArgLength = ((RasGlobalStorage *)thr->javaVM->j9rasGlobalStorage)->methodStrArgLength;
 
-		if (clazz == J9VMJAVALANGSTRING_OR_NULL(vm)) {
+		if (clazz == J9VMJAVALANGSTRING_OR_NULL(vm) &&
```

Comment:
The && should be on the left.

```
if (clazz == J9VMJAVALANGSTRING_OR_NULL(vm) 
&& (0 != methodStrArgLength)
) {
```

see https://github.com/eclipse-omr/omr/blob/master/doc/CodingStandard.md#formatting-conditions